lib/modules/manager/rust-toolchain/extract.ts

```diff
@@ -0,0 +1,23 @@
+import { regEx } from '../../../util/regex';
+import { GithubReleasesDatasource } from '../../datasource/github-releases';
+import { regexMatchAll } from '../custom/regex/utils';
+import type { PackageDependency, PackageFileContent } from '../types';
+
+const channelRegex = regEx(/channel\s*=\s*"(?<currentValue>.+)"/, 'g');
+
+export function extractPackageFile(content: string): PackageFileContent {
+  const deps: PackageDependency[] = [];
+  for (const match of regexMatchAll(channelRegex, content)) {
```

Comment:
theres only one occurrence, so need of match all

I would prefer to use our toml parser and zod schema validation. so we don't have false deps when they extend the format